KarhooUISDK.xcodeproj/project.pbxproj

```diff
@@ -366,6 +366,11 @@
 		40FE0CF022D7D31A003F65B5 /* KarhooRatingView.swift in Sources */ = {isa = PBXBuildFile; fileRef = 40FE0CEF22D7D31A003F65B5 /* KarhooRatingView.swift */; };
 		40FE0CF222D7D333003F65B5 /* KarhooRatingPresenter.swift in Sources */ = {isa = PBXBuildFile; fileRef = 40FE0CF122D7D333003F65B5 /* KarhooRatingPresenter.swift */; };
 		511D9CCE27D90193004CBA6B /* UINAvigationController+navigationCompletions.swift in Sources */ = {isa = PBXBuildFile; fileRef = 511D9CCD27D90193004CBA6B /* UINAvigationController+navigationCompletions.swift */; };
+		5132EAF627EBA6F100866FA9 /* MainActionButton.swift in Sources */ = {isa = PBXBuildFile; fileRef = 5132EAF527EBA6F000866FA9 /* MainActionButton.swift */; };
+		5132EAF927ECD02500866FA9 /* SingleSelectionListView.swift in Sources */ = {isa = PBXBuildFile; fileRef = 5132EAF827ECD02500866FA9 /* SingleSelectionListView.swift */; };
+		5132EAFC27ECF5E100866FA9 /* UserSelectable.swift in Sources */ = {isa = PBXBuildFile; fileRef = 5132EAFB27ECF5E100866FA9 /* UserSelectable.swift */; };
+		5132EB0227F1E1D000866FA9 /* RadioControl.swift in Sources */ = {isa = PBXBuildFile; fileRef = 5132EB0127F1E1D000866FA9 /* RadioControl.swift */; };
+		5132EB0427F2517B00866FA9 /* UIControl+addTouchAnimation.swift in Sources */ = {isa = PBXBuildFile; fileRef = 5132EB0327F2517B00866FA9 /* UIControl+addTouchAnimation.swift */; };
```

Comment:
I think `addTouchAnimation` was meant to be `AddTouchAnimation` with a capital letter, right?